domenetjenester/vedtak/src/main/java/no/nav/foreldrepenger/domene/vedtak/migrering/OpprettManglendeForespørslerTask.java

```diff
@@ -0,0 +1,96 @@
+import no.nav.foreldrepenger.behandlingslager.fagsak.FagsakProsesstaskRekkefølge;
+import no.nav.foreldrepenger.behandlingslager.fagsak.FagsakStatus;
+import no.nav.foreldrepenger.behandlingslager.fagsak.FagsakYtelseType;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTask;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskData;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskHandler;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskTjeneste;
+
+@ApplicationScoped
+@ProsessTask(value = "fpinntektsmelding.opprettManglendeforespørsler", prioritet = 3, maxFailedRuns = 1)
+@FagsakProsesstaskRekkefølge(gruppeSekvens = false)
+public class OpprettManglendeForespørslerTask implements ProsessTaskHandler {
+    private static final Logger LOG = LoggerFactory.getLogger(OpprettManglendeForespørslerTask.class);
+    private static final String FRA_FAGSAK_ID = "fraFagsakId";
+    private static final String TOM_FAGSAK_ID = "tomFagsakId";
+    private static final String DRY_RUN = "dryRun";
+
+    private EntityManager entityManager;
+    private ProsessTaskTjeneste prosessTaskTjeneste;
+    private MigrerManglendeForespørslerTjeneste migrerManglendeForespørslerTjeneste;
+
+    OpprettManglendeForespørslerTask() {
+        // for CDI proxy
+    }
+
+    @Inject
+    public OpprettManglendeForespørslerTask(EntityManager entityManager,
+                                            ProsessTaskTjeneste prosessTaskTjeneste,
+                                            MigrerManglendeForespørslerTjeneste migrerManglendeForespørslerTjeneste) {
+        this.entityManager = entityManager;
+        this.prosessTaskTjeneste = prosessTaskTjeneste;
+        this.migrerManglendeForespørslerTjeneste = migrerManglendeForespørslerTjeneste;
+    }
+
+    @Override
+    public void doTask(ProsessTaskData prosessTaskData) {
+        var fraFagsakId = Optional.ofNullable(prosessTaskData.getPropertyValue(FRA_FAGSAK_ID)).map(Long::valueOf).orElseThrow();
+        var tomFagsakId = Optional.ofNullable(prosessTaskData.getPropertyValue(TOM_FAGSAK_ID)).map(Long::valueOf).orElse(2188628L);
+        var dryRun = Boolean.parseBoolean(Optional.ofNullable(prosessTaskData.getPropertyValue(DRY_RUN)).orElse("true"));
+
+        var saker = finnNesteHundreSaker(fraFagsakId, tomFagsakId);
+
+        saker.forEach(sak -> {
+            migrerManglendeForespørslerTjeneste.vurderOmForespørselSkalOpprettes(sak, dryRun);
+        });
+
+        saker.stream()
+            .map(Fagsak::getId)
+            .max(Long::compareTo)
+            .ifPresent(sisteId -> prosessTaskTjeneste.lagre(opprettManglendeForespørselTaskForNesteDato(sisteId+1, dryRun)));
+    }
+
+    private List<Fagsak> finnNesteHundreSaker(Long nesteFagsakId, Long tomFagsakId) {
+        var sql = """
+            select * from (
+            select fag.* from FAGSAK fag
+            where fag.id >= :fraFagsakId and fag.id <= :tomFagsakId
+           and fag.opprettet_tid < :nyImPortalLansertDato
```

Comment:
Trenger du sjekken på opprettet_tid? Regner med at defaultverdi for tomFagsakId er funnet fra databasen .